woocommerce-paypal-payments.php

```diff
@@ -81,14 +81,23 @@ function() {
 		'plugins_loaded',
 		function () {
 			init();
+
+			if ( ! function_exists( 'get_plugin_data' ) ) {
+				require_once ABSPATH . 'wp-admin/includes/plugin.php';
+			}
+			$plugin_data    = get_plugin_data( __DIR__ . '/woocommerce-paypal-payments.php' );
+			$plugin_version = $plugin_data['Version'] ?? null;
+			if ( get_option( 'woocommerce-ppcp-upgrading-version' ) !== $plugin_version ) {
+				do_action( 'woocommerce_paypal_payments_gateway_migrate' );
+				update_option( 'woocommerce-ppcp-upgrading-version', $plugin_version );
```

Comment:
maybe it should be just `woocommerce-ppcp-version`?

Reply:
Done :)